lib/style-formatters/css.js

```diff
@@ -46,19 +49,30 @@ module.exports = (symbols = [], options = {}, sources = []) => {
             svg.appendChild(childNode);
         });
 
+        // Extract width/height from viewbox
+        const size = symbol.getAttribute('viewBox').split(' ').slice(2);
+        const width = options.units === 'rem' ? `${pxToRem(size[0])}rem` : `${size[0]}px`;
+        const height = options.units === 'rem' ? `${pxToRem(size[1])}rem` : `${size[1]}px`;
+
         const prefix = generateSpritePrefix(options.prefix, sources[index].path);
         const selector = symbol.getAttribute('id').replace(prefix, '').replace(options.postfix.symbol, '');
-        const content = ((svg) => {
+        let content = ((svg) => {
             switch ( options.format.type ) {
                 case 'data':
-                    return svgToMiniDataURI(XMLSerializer.serializeToString(svg));
+                    return ` background-image: url("${svgToMiniDataURI(XMLSerializer.serializeToString(svg))}");`;
                 case 'fragment':
                     const postfix = typeof options.postfix.view === 'boolean' ? '' : options.postfix.view;
-                    return `${options.format.publicPath}#${prefix}${selector}${postfix}`;
+                    return ` background-image: url("${options.format.publicPath}#${prefix}${selector}${postfix}");`;
```

Comment:
It probably would be better to return the value and keep `background-image:` where it was